src/libexpr/eval.hh

```diff
@@ -835,11 +835,9 @@ struct EvalSettings : Config
     Setting<bool> enableImportFromDerivation{
         this, true, "allow-import-from-derivation",
         R"(
-          By default, Nix allows you to `import` from a derivation, allowing
-          building at evaluation time. With this option set to false, Nix will
-          throw an error when evaluating an expression that uses this feature,
-          allowing users to ensure their evaluation will not require any
-          builds to take place.
+          By default, Nix allows [Import from Derivation](@docroot@/language/import-from-derivation.md).
+          With this option set to `false`, Nix will throw an error when evaluating an expression that uses this feature,
+          ensuring that evaluation will not require any builds to take place.
```

Comment:
```suggestion
          ensuring that the evaluated expressions will not require any builds to take place, regardless of the state of the store.
```

I think you were trying to say that it establishes a property reproducibly.

```suggestion
          even when the required store object is readily available,
          ensuring that evaluation will not require any builds to take place,
          regardless of the state of the store.
```

We could also mention pure-eval and call it reproducible, but that feels like too much info.

Reply:
Open for suggestions to mention `pure-eval`. Not sure what that would look like or how exactly it even relates.